tests/playbooks/tests_bond.yml

```diff
@@ -19,6 +19,10 @@
     - import_tasks: tasks/assert_device_present.yml
       vars:
         interface: "{{ dhcp_interface2 }}"
+    - name: "Backup the /etc/resolv.conf for initscript"
+      command: cp -vf /etc/resolv.conf /etc/resolv.conf.bak
+      when:
+        - network_provider == "initscripts"
```

Comment:
I think this task needs a `changed_when: false` but I'm not sure why it wasn't flagged by ansible-lint

Reply:
Why `changed_when` is required here?

Reply:
I guess it doesn't matter since this is in test code, and so will always report `changed` for the task, and will never be run again, so it won't run afoul of idempotency.